airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -596,6 +589,18 @@ def add_to_outputs(self, sql: str, is_intermediate: bool, suffix: str = "") -> s
         print(f"  Generating {output} from {json_path}")
         return ref_table(file_name)
 
+    def get_file_name(self, stream_name: str, schema_name: str, table_name: str) -> str:
+        """
+        File names need to match the ref() macro returned in the ref_table function.
+        Note that dbt uses only the file names to generate internal model.
+
+        Use a hash of full schema + stream name (i.e. namespace + stream name) to dedup tables.
+        This hash avoids tables with the same name (possibly very long) and different schemas.
+        """
+        full_lower_name = schema_name + "_" + stream_name
+        full_lower_name = full_lower_name.lower()
+        return self.name_transformer.normalize_table_name(f"{table_name}_{hash_name(full_lower_name)}", False, False)
+
```

Comment:
the dbt model file_name is now following:
- <table_name>_<hash of schema + stream name>.sql 

instead of 
- <schema_name>_<table_name>.sql

Reply:
Makes sense to me.